src/main/kotlin/com/chattriggers/ctjs/CTJS.kt

```diff
@@ -26,8 +27,13 @@ import net.minecraftforge.fml.common.event.FMLPreInitializationEvent
 import java.io.File
 import java.net.URL
 import java.net.URLConnection
+import java.security.KeyStore
+import javax.net.ssl.HttpsURLConnection
```

Comment:
These imports don't look sorted; is this how IntelliJ imported them?

Reply:
It is, is there a specific convention for sorting imports I should follow?